sonar-slang-plugin/src/test/java/com/sonarsource/slang/kotlin/KotlinConverterTest.java

```diff
@@ -139,6 +139,20 @@ public void testFunctionDeclaration() {
     assertTree(aStringParam).isNotEquivalentTo(bStringParam);
   }
 
+  @Test
+  public void testExtensionFunction() {
+    Tree extensionFunction1 = kotlin("fun A.fun1() {}");
+    Tree extensionFunction2 = kotlin("fun B.fun1() {}");
+    Tree normalFunction = kotlin("fun fun1() {}");
+    Tree extensionFunction3 = kotlin("fun A.fun1() {}");
+    Tree memberFunction = kotlin("class A { fun fun1() {} }");
+
+    assertTree(extensionFunction1).isNotEquivalentTo(extensionFunction2);
+    assertTree(extensionFunction1).isNotEquivalentTo(normalFunction);
+    assertTree(extensionFunction1).isEquivalentTo(extensionFunction3);
+    assertTree(extensionFunction1).isNotEquivalentTo(memberFunction);
```

Comment:
Minor: I would have inlined all variables in this test, it would be more readable to me.